internal/app/api/routes/routes.proto

```diff
@@ -0,0 +1,411 @@
+// Routes for wwapi.
+// TODO: Try protoc-gen-doc for generating documentation.
+// Note: We may want rpcs for Entry accessors, but we may not need them. Unclear.
+syntax = "proto3";
+
+option go_package = "github.com/hpcng/warewulf/tree/main/internal/pkg/api/routes";
```

Comment:
I think you can just specify `internal/pkg/api/routes` here and then the branch won't be included so it will import as `github.com/hpcng/warewulf/internal/pkg/api/routes`.